paymentsheet/src/main/java/com/stripe/android/link/repositories/LinkApiRepository.kt

```diff
@@ -61,6 +61,12 @@ internal class LinkApiRepository @Inject constructor(
     private val fraudDetectionDataRepository: FraudDetectionDataRepository =
         DefaultFraudDetectionDataRepository(application, workContext)
 
+    private val apiRequestOptions: ApiRequest.Options
```

Comment:
suggestion:
 1. prefix with "default" since we have `buildRequestOptions()`
 2. cache instead of using a getter to avoid recreating the same object

```suggestion
    private val defaultRequestOptions: ApiRequest.Options =
        ApiRequest.Options(
            publishableKeyProvider = publishableKeyProvider,
            stripeAccountIdProvider = stripeAccountIdProvider,
        )
```

Reply:
Oof, that's super confusing. We should delete that constructor and require the caller to just invoke the provider functions (not necessarily in this PR). Could we reuse `buildRequestOptions()` then?
```kotlin
    private val defaultRequestOptions: ApiRequest.Options
        get() = buildRequestOptions()
```